src/style.css

```diff
@@ -0,0 +1,124 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+.search-field {
+  box-sizing: border-box;
+  display: block;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  height: 70px;
+  margin: 20px 0;
+}
+
+.input-of-field {
+  position: absolute;
+  border: none;
+  outline: none;
+  margin-top: 23px;
+  padding: 0;
+  height: 22px;
+  font-family: inherit;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  margin-left: 18px;
+}
+
+.image-of-field {
+  margin-top: 24px;
+  margin-left: 25px;
+}
+
+.input-of-field::placeholder {
+  font-family: inherit;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.search-field:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.search-field:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-family: inherit;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.search-field--small {
+  box-sizing: border-box;
+  position: relative;
+  display: block;
+  height: 42px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.input-of-field--small {
```

Comment:
create common classes for big and small forms, move repetitive styles there to prevent code duplicating